### PR TITLE
Feat/alarm classification and notify channel

### PR DESCRIPTION
**Pull request by sherotree**
## What this PR does / why we need it:
1. add notice-channel page 
image

2. in alarm strategy
add trigger conditions,modify notifies, add alarm level and isRecover to rules
 image
image

## Does this PR introduce a user interface change?

✅ Yes(screenshot is required)

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |    1. add the notice-channel page to orgCenter  2.  in alarm strategy,add trigger conditions, modify notifies, add alarm level, and isRecover to rules |
| 🇨🇳 中文    |      1、管理中心中增加通知渠道  2、在告警策略中增加触发条件、更改告警策略，并在规则中增加告警级别字段与触发恢复字段      |

## Does this PR need be patched to older version?
✅ Yes(version is required)
release/1.4

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/modules/cmp/common/alarm-strategy/index.tsx [hunk 1]
```diff
@@ -71,6 +75,58 @@ const notifyGroupPage = {
   [ScopeType.MSP]: goTo.pages.mspProjectNotifyGroup,
 };
 
+const alertLevelOptions = [
+  {
+    key: 'Panic',
+    display: i18n.t('msp:breakdown'),
```
Comment (daskyrk) [inline shell/app/modules/cmp/common/alarm-strategy/index.tsx:81]: should same with key

File: shell/app/modules/cmp/common/alarm-strategy/index.tsx [hunk 2]
```diff
@@ -71,6 +75,58 @@ const notifyGroupPage = {
...
+  {
+    key: 'Light',
+    display: i18n.t('msp:light'),
+  },
+];
+
+const conditionOperatorOptions = [
+  {
+    key: 'like',
+    display: i18n.t('msp:match'),
```
Comment (daskyrk) [inline shell/app/modules/cmp/common/alarm-strategy/index.tsx:100]: should same with key

File: shell/app/modules/cmp/common/alarm-strategy/index.tsx [hunk 3]
```diff
@@ -71,6 +75,58 @@ const notifyGroupPage = {
...
+    type: 'input',
+  },
+  {
+    key: 'not like',
+    display: i18n.t('msp:not match'),
```
Comment (daskyrk) [inline shell/app/modules/cmp/common/alarm-strategy/index.tsx:105]: same

File: shell/app/modules/cmp/common/alarm-strategy/notify-strategy-select.tsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+interface IProps {
+  id: string;
+  handleEditNotifyStrategy: (id: string, item: { key: string; value: string }) => void;
+  valueOptions: Array<{ key: string; display: string }>;
+  addNotificationGroupAuth: boolean;
+  goToNotifyGroup: () => void;
+  notifyGroups: COMMON_NOTIFY.INotifyGroup[];
+  alertLevelOptions: Array<{ key: string; display: string }>;
+  notifyChannelMap: typeof notifyChannelOptionsMap | typeof smsNotifyChannelOptionsMap;
+  handleRemoveNotifyStrategy: (id: string) => void;
```
Comment (daskyrk) [inline shell/app/modules/cmp/common/alarm-strategy/notify-strategy-select.tsx:35]: missing updater type

File: shell/app/modules/cmp/services/alarm-strategy.ts [hunk 5]
```diff
@@ -64,3 +64,19 @@ export const getClusterList = ({ orgId }: { orgId: number }) => {
...
+  return agent
+    .get('/api/cmp/alerts/conditions')
+    .query({ scopeType })
+    .then((response: any) => response.body);
+};
+
+export const getAlertTriggerConditionsContent = (
+  params: COMMON_STRATEGY_NOTIFY.IAlertTriggerConditionQuery,
+): COMMON_STRATEGY_NOTIFY.IAlertTriggerConditionContent[] => {
+  return agent
```
Comment (daskyrk) [inline shell/app/modules/cmp/services/alarm-strategy.ts:78]: Why not use api-creator

Comment (sherotree) [reply shell/app/modules/cmp/services/alarm-strategy.ts:78]: For MSP and CMP use this component together, not use apiCreator in this file

**Pull request merged**